content/en/blog/2022/meet-our-contributors-APAC-China-region-03.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: blog
+title: "Meet Our Contributors - APAC (China region)"
+date: 2022-07-11
```

Comment:
How about we update this article (and its mirror) to publish on August 15th?

Reply:
@sftim Sure. Some suggestions on the k/website PR are still not resolved, so I was waiting for the SIG Docs members to comment on those but it's been weeks now. I'll make some minor changes and prepare the article and the mirror to get published on August 15th now.